src/main/java/com/arpnetworking/metrics/impl/ApacheHttpSink.java

```diff
@@ -119,8 +127,14 @@ private ApacheHttpSink(final Builder builder) {
 
     private final class HttpDispatch implements Runnable {
 
-        /* package private */ HttpDispatch(final PoolingHttpClientConnectionManager connectionManager) {
-            _connectionManager = connectionManager;
+        /* package private */ HttpDispatch(
+                final PoolingHttpClientConnectionManager connectionManager,
+                final Duration dispatchErrorLoggingInterval) {
+                _connectionManager = connectionManager;
+             _dispatchErrorLogger = new RateLimitedLogger(
```

Comment:
I don't think this is quite right.  You're going to create one of these on each thread.  Your actual rate will be multiplied by the number of threads running.